cmd/nitro/init.go

```diff
@@ -332,6 +332,30 @@ func dirExists(path string) bool {
 	return info.IsDir()
 }
 
+func checkEmptyDatabaseDir(dir string, force bool) error {
+	entries, err := os.ReadDir(dir)
+	if err != nil {
+		return fmt.Errorf("failed to open database dir %s: %w", dir, err)
+	}
+	unexpectedFiles := []string{}
+	allowedFiles := map[string]bool{
+		"LOCK": true, "classic-msg": true, "l2chaindata": true,
+	}
+	for _, entry := range entries {
+		if !allowedFiles[entry.Name()] {
+			unexpectedFiles = append(unexpectedFiles, entry.Name())
+		}
+	}
+	if len(unexpectedFiles) > 0 {
+		if force {
```

Comment:
obs: I think it is a little bit weird that a force argument would generate an error instead of overwriting.
This is was not introduced in this PR, so just adding an observation.

Reply:
That was the original intent. The idea is to avoid deleting files that nitro might not have created.

Reply:
:+1: